Dockerfile

```diff
@@ -0,0 +1,16 @@
+
+FROM astral/uv:bookworm-slim
+
+
+RUN mkdir /app
+RUN mkdir /app/run
+WORKDIR /app
+COPY pyproject.toml ./
+COPY uv.lock ./
+COPY README.md ./
+
+COPY carrottransform/ ./carrottransform/
+RUN uv run python -m carrottransform.cli.command run mapstream --help
+
+
+ENTRYPOINT ["uv", "run", "python", "-m", "carrottransform.cli.command", "run", "mapstream"]
```

Comment:
This means this we are hardcoded to always run the `mapstream ` command, which won't allow for example the V2 command? 

It's worth using the `CMD` instead so it a user can override it